resources/web/streaming_viewer/index.html

```diff
@@ -5,14 +5,16 @@
     <title>Streaming viewer</title>
     <link rel='stylesheet' href='https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css'/>
     <link type="text/css" rel="stylesheet" href='https://www.cyberbotics.com/jquery-ui-layout/1.4.4/layout-default.css' />
-    <link type="text/css" rel="stylesheet" href='https://www.cyberbotics.com/wwi/R2019a/wwi.css' />
+    <link type="text/css" rel="stylesheet" href='../wwi/wwi.css' />
     <link type="text/css" rel="stylesheet" href='style.css' />
     <script src='https://code.jquery.com/jquery-3.1.1.min.js' integrity='sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=' crossorigin='anonymous'></script>
     <script src='https://code.jquery.com/ui/1.12.1/jquery-ui.min.js' integrity='sha256-VazP97ZCwtekAsvgPBSUwPFKdrwD3unUfSGVYrahUqU=' crossorigin='anonymous'></script>
     <script src='https://www.cyberbotics.com/jquery-ui-layout/1.4.4/jquery.layout.js'></script>
     <script src='https://www.cyberbotics.com/jquery-dialogextend/2.0.4/jquery.dialogextend.min.js'></script>
     <script src='https://cdn.jsdelivr.net/ace/1.2.6/min/ace.js'></script>
-    <script src="https://www.cyberbotics.com/wwi/R2019a/webots.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/three.js/100/three.js"></script>
```

Comment:
Why not adding the GPUPicker dependencies (and even THREEjs?) inside terser?

Reply:
We can discuss this during the meeting.
But if we do it for THREEjs, GPUPicker, why not doing it as well for all the other dependencies?

Reply:
Yes, we should choose between everything or nothing :-)